includes/Classifai/Providers/Azure/Read.php

```diff
@@ -125,7 +125,20 @@ public function read_document() {
 
 		$filesize = filesize( get_attached_file( $this->attachment_id ) );
 		if ( ! $filesize || $filesize > computer_vision_max_filesize() ) {
-			return $this->log_error( new WP_Error( 'size_error', esc_html__( 'Document does not meet size requirements. Please ensure it is smaller than the maximum threshold (defaults to 4MB).', 'classifai' ), $metadata ), $attachment_id );
+			return $this->log_error(
+				new WP_Error(
+					'size_error',
+					esc_html(
+						sprintf(
+							// translators: %1$s is the document file size, %2$s is the current default max filesize
+							__( 'Document (%1$s) does not meet size requirements. Please ensure it is smaller than the maximum threshold (currently %1$s, defaults to 4MB).', 'classifai' ),
```

Comment:
Might be nice to make it clear that the `$filesize` and max file size values are in bytes, since we show the default here in MB (or convert those sizes into MB so they all match)

Reply:
Updated in https://github.com/10up/classifai/pull/443/commits/f9e5aa69f16eacbd58d117704fb4730a460b4083 to use bytes for all sizes.